### PR TITLE
first integration test for CountReads

### PR DESCRIPTION
**Pull request by jean-philippe-martin**

### CODE REVIEW

**Comment by jean-philippe-martin** [comment]
Adding a test decreases the coverage? Thie coveralls report doesn't make much sense to me.

File: src/main/java/com/google/cloud/genomics/dataflow/pipelines/CountReads.java [hunk 1]
```diff
@@ -140,7 +141,7 @@ public void processElement(DoFn<Long, String>.ProcessContext c) throws Exception
         c.output(String.valueOf(c.element()));
       }
     }).named("toString"));
-    readCountText.apply(TextIO.Write.to(options.getOutput()).named("WriteOutput"));
+    readCountText.apply(TextIO.Write.to(options.getOutput()).named("WriteOutput").withoutSharding());
```
Comment (deflaux) [inline src/main/java/com/google/cloud/genomics/dataflow/pipelines/CountReads.java]: Please undo this change and add `--shardBAMReading=false` to the options in the test.

**Comment by deflaux** [comment]
This is great!

File: src/main/java/com/google/cloud/genomics/dataflow/pipelines/CountReads.java (hunk 1 above)
Comment (jean-philippe-martin) [inline src/main/java/com/google/cloud/genomics/dataflow/pipelines/CountReads.java]: Those aren't the same thing: --shardBAMReading changes how the BAM file is read (sharded or not). This line changes how the output file is written.

Comment (deflaux) [inline src/main/java/com/google/cloud/genomics/dataflow/pipelines/CountReads.java]: gotcha, thanks for the clarification!

**Comment by deflaux** [comment]
Regarding the test coverage decrease, we should take a closer look at the coveralls configuration.  I'm not sure whether coveralls is specific to unit tests.
